src/auth.js

```diff
@@ -0,0 +1,66 @@
+const inquirer = require('inquirer')
```

Comment:
I see you have found one! 👍

Reply:
This one is just for "cool" and "sexy" user interactions like wizard. For handling CLI there is `commander`